### PR TITLE
Update API

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: src/clients/Queue.js [hunk 1]
```diff
@@ -38,9 +38,10 @@ export default class Queue extends Client {
...
     this.declareWorkerType.entry = {type:'function',method:'put',route:'/provisioners/<provisionerId>/worker-types/<workerType>',query:[],args:['provisionerId','workerType'],name:'declareWorkerType',stability:'experimental',scopes:[['queue:declare-worker-type:<provisionerId>/<workerType>#<property>']],input:true,output:true}; // eslint-disable-line
-    this.listWorkers.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types/<workerType>/workers',query:['continuationToken','limit','disabled'],args:['provisionerId','workerType'],name:'listWorkers',stability:'experimental',output:true}; // eslint-disable-line
+    this.listWorkers.entry = {type:'function',method:'get',route:'/provisioners/<provisionerId>/worker-types/<workerType>/workers',query:['continuationToken','limit','quarantined'],args:['provisionerId','workerType'],name:'listWorkers',stability:'experimental',output:true}; // eslint-disable-line
```
Comment (eliperelman) [inline src/clients/Queue.js]: This looks like a breaking change, renaming the query property. @djmitche?

Comment (helfi92) [reply src/clients/Queue.js]: The front-end has already been changed to reflect this endpoint: https://github.com/taskcluster/taskcluster-tools/blob/0c839a2718f190c6dd38029030c3a9896f2d2f54/src/views/Workers/WorkerManager.js#L115.

listWorkers will have the `quarantined` query when a user filters on quarantined workers through the filter dropdown in the workers page. In the default behaviour when loading the workers page, the `quarantined` property is not set.

Comment (djmitche) [reply src/clients/Queue.js]: Yes, it was a breaking change to the API, but it's still experimental so nbd :)

File: src/clients/AwsProvisioner.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Client from '../Client';
 export default class AwsProvisioner extends Client {
   constructor(options = {}) {
     super({
-      baseUrl: 'https://aws-provisioner.taskcluster.net/v1',
+      baseUrl: 'https://staging-aws-provisioner.herokuapp.com/v1',
```
Comment (djmitche) [inline src/clients/AwsProvisioner.js]: This, however, is problematic!  It looks like the [staging provisioner has published to the production bucket](http://references.taskcluster.net/aws-provisioner/v1/exchanges.json).  @jhford do you want to take a look?

**Comment by eliperelman** [comment]
Published at v5.0.0.

**Pull request merged**